parse-pkgs.sh

```diff
@@ -54,6 +54,8 @@ INSTALLER_TAG=$(linuxkit_tag pkg/installer)-$ARCH
 # images lacking functionality.
 ZTOOLS_TAG=${ZTOOLS_TAG:-$(plugin_tag zededa/ztools:latest)}
 
+[ $# -eq 1 ] && INFILE=$1
+
```

Comment:
I actually don't think you need this part. If no arguments are specified, $1 is empty which is supposed to do exactly the right thing with sed. Can you please remove the change to parse-pkgs.sh and only leave the Makefile change in?

Reply:
Right, missed that (iirc, something to that effect was removed earlier, so it's a bit deja-vu). 

I've reworked it from the ex-zededa id, but since I no longer have access to the zededa git id/email, this pull request will have to be abandoned. I'll raise separate ones from the other id which I'm commenting via.

Reply:
Sounds good! Waiting for the other one to show up!